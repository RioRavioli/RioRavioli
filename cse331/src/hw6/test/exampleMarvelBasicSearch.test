# Load the Marvel dataset and search for an existing path.
# To run this test, you will need to have downloaded the marvel dataset and
# place it in hw6/test/marvel.tsv.

LoadGraph marvel marvel.tsv

FindPath marvel BEAST/HENRY_&HANK&_P WISDOM,_PETER
