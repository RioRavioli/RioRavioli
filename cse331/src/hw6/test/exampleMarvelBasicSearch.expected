# Load the Marvel dataset and search for an existing path.
# To run this test, you will need to have downloaded the marvel dataset and
# place it in hw6/test/marvel.tsv.

loaded graph marvel

path from BEAST/HENRY &HANK& P to WISDOM, PETER:
BEAST/HENRY &HANK& P to CYCLOPS/SCOTT SUMMER with weight 0.003
CYCLOPS/SCOTT SUMMER to WOLVERINE/LOGAN  with weight 0.004
WOLVERINE/LOGAN  to SHADOWCAT/KATHERINE  with weight 0.006
SHADOWCAT/KATHERINE  to WISDOM, PETER with weight 0.026
total cost: 0.039
